traffic_ops/app/lib/Schema/Result/DeliveryServiceInfoForCdnList.pm

```diff
@@ -0,0 +1,103 @@
+
+
+#
+# Copyright 2015 Comcast Cable Communications Management, LLC
```

Comment:
This one isn't generated.  There are a few others in there like that - it returns specific results.